test/v1_2_0/unit/escrow/escrow/EscrowWithdraw.t.sol

```diff
@@ -38,6 +38,20 @@ contract TestWithdraw is IEscrowCurveTokenStorage, IGaugeVote, ITicket, EscrowBa
         escrow.setMinDeposit(0);
     }
 
+    function testRevertIfCreateLockAndBeginWithdrawalInSameTx() public {
```

Comment:
nice - is there a test where we check that something like:

create+merge
create+split
create+beginWithdrawal

All:

- Can't be called in the same block
- If called in the same checkpoint interval correctly adjust the state variables in the curve